OSDOCS-7799: Migrated a topic about using Azure AD for group claims (#65529)
openshift/openshift-docs: cloud_experts_tutorials/cloud-experts-entra-id-idp.adoc

:_content-type: ASSEMBLY
[id="cloud-experts-entra-id-idp"]
= Tutorial: Configuring Microsoft Entra ID (formerly Azure Active Directory) as an identity provider
include::_attributes/attributes-openshift-dedicated.adoc[]
:context: cloud-experts-entra-id-idp

toc::[]

//Mobb content metadata
//Brought into ROSA product docs 2023-09-18
// ---
// date: '2022-09-19'
// title: Configure ROSA to use Entra ID Group Claims
// authors:
//   - Michael McNeill
//   - Paul Czarkowski
// ---

> This guide demonstrates how to configure Microsoft Entra ID (formerly Azure Active Directory) as the cluster identity provider in {product-title} (ROSA). This guide will walk through the creation of an Microsoft Entra ID (Entra ID) application and configure Red Hat OpenShift Service on AWS (ROSA) to authenticate using Azure AD. 

Reviewer: ```suggestion
Use this tutorial to configure Microsoft Entra ID (formerly Azure Active Directory) as the cluster identity provider in {product-title} (ROSA). This tutorial will walk through creating a Microsoft Entra ID (Entra ID) application and configuring ROSA to authenticate using Azure AD. 
```

> This guide will walk through the following steps:

Reviewer: ```suggestion
This tutorial walks you through the following steps:
```
